src/idpf.rs

```diff
@@ -158,7 +158,7 @@ pub trait IdpfValue:
     + Sized
 {
     /// Any run-time parameters needed to produce a value.
-    type ValueParameter;
+    type ValueParameter: Clone + std::fmt::Debug;
```

Comment:
I removed a couple, but most of the bounds that were added were to subfields of the Mastic struct, which has to be Debug to fit the VDAF trait. And I think it will be useful for many of them (VidpfKey, VidpfValues) to be Debuggable, especially since we don't need any extra implementations since FieldElements already implement Debug.